crates/oxc_language_server/src/main.rs

```diff
@@ -359,6 +359,8 @@ impl Backend {
                     Oxlintrc::from_file(&config_path)
                         .expect("should have initialized linter with new options"),
                 )
+                // FIXME: Handle this error more gracefully and report it properly
```

Comment:
Let's not crash the IDE 😅 shall we silence the error before the fix?

Reply:
I think it's probably okay for now, since we already have two other unwrap/expect calls in this function? I need more expertise on the editor side though, I feel like the "right" way to handle this is to create an error popup, but maybe that happens automatically?

Reply:
Yeah the correct way is to show a popup, I don't know what happens when it panics.

Reply:
I can test it out and see what happens

Reply:
From memory, I believe this is the behavior for VS Code and IntelliJ.
The language server will crash. If it crashes multiple times, a pop up will display indicating it has been disabled. You can restart your IDE to reenable it. You can check your logs to see what might have happened. (Oxc channel in VS Code or idea.log for IntelliJ)

If you want a popup, that needs to be communicated by the language server with a message to the IDE. I don’t know if a standard message exists, or if this would be a custom one.

For now, if you just catch the error and log it to stderr then I can fix the editor support after.